parkings/api/operator/parking.py

```diff
@@ -49,6 +49,12 @@ def __init__(self, *args, **kwargs):
 
         self._set_required_extra_fields()
 
+        domain = self.context['request'].user.enforcer.enforced_domain
```

Comment:
The user is not supposed to be an Enforcer here. It's an operator. Use the domain of the parking as the domain on this line. Though it probably can't be done like this, since the parking is not yet created/initialized.

Reply:
How can we currently know when posting the new parking what the domain should be?

Reply:
Already replied in Slack, but repeating here for future reference. It's supposed to be part of the parameters in the request, or if it's left out, the default value is from settings (as used by EnforcementDomain.get_default_domain function).